Tasks/PublishBuildArtifactsV1/publishbuildartifacts.ts

```diff
@@ -43,13 +44,51 @@ let pathToRobocopyPSString = (filePath: string) => {
     return `'${result}'`;
 }
 
+function createTarArchive(filesPath: string, artifactName: string) {
```

Comment:
Nit: would be great to have JS Docs also 😃

Reply:
Yeah, added them here too